### PR TITLE
Add PyMappingProtocol impl for PyEmbeddings.

### PR DESCRIPTION
**Pull request by sebpuetz**

### CODE REVIEW

File: src/embeddings.rs
Comment (danieldk) [inline src/embeddings.rs]: Since we now look up embeddings in two places, this could be factored out to a method in `EmbeddingsWrap`.

File: tests/test_embedding.py
Comment (danieldk) [inline tests/test_embedding.py]: Use:

```python
with pytest.raises(KeyError):
    embeds["Something out of vocabulary"]
```

Also, with #40, leading the embeddings here is not necessary anymore ;).

**Review by danieldk — CHANGES_REQUESTED** [review]
Two small suggestions for improvement.

**Comment by sebpuetz** [comment]
I'll make the changes and rebase once #40 is merged

**Comment by danieldk** [comment]
> I'll make the changes and rebase once #40 is merged

#40 is merged now

**Comment by sebpuetz** [comment]
> > I'll make the changes and rebase once #40 is merged
> 
> #40 is merged now

I already rebased and pushed the changes

**Review by danieldk — APPROVED** [review]
Looks good, will merge after CI acks.

**Pull request merged**
